README.md

```diff
@@ -3,3 +3,64 @@
 Welcome to the Void documentation. This repository contains the source data
 behind <https://docs.voidlinux.org/>. Contributing to this repository follows
 the same protocol as the packages tree.
+
+## Contributing
+
+### Requirements
+
+To clone the repository and push changes you need git installed.
+
+```
+# xbps-install -S git
+```
+
+Building the Void Handbook locally requires
+[mdBook](https://rust-lang-nursery.github.io/mdBook/) to be installed. To get
+it:
```

Comment:
which can be installed with the `mdBook` package on Void.

(we're trying to avoid using the package install commands all over the place)

Reply:
1. > Building the Void Handbook locally requires the [`mdBook`](https://rust-lang-nursery.github.io/mdBook/) package to be installed.

or

2. > Building the Void Handbook locally requires [mdBook](https://rust-lang-nursery.github.io/mdBook/), which can be installed with the `mdBook` package on Void.

I like 1. better because it's more concise, but not the link inside the code. The link could be dropped. 2. Doesn't have that issue but "mdBook" gets duplicated.

Reply:
I prefer 2 because it mentions mdBook with a link to the project to provide more information, then mentions the package explicitly.

Reply:
It might also be useful to mention running `mdbook serve` in the root of the repository because that is the best workflow for editing the docs. I've never 'built' the book statically while working on it.